src/main/java/edu/ucsb/cs156/dining/repositories/ReviewRepository.java

```diff
@@ -0,0 +1,27 @@
+package edu.ucsb.cs156.dining.repositories;
+
+import edu.ucsb.cs156.dining.entities.Review;
+
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+/**
+ * The RestaurantRepository is a repository for Restaurant entities
```

Comment:
It's a what now?

Be careful when copying/pasting code that you adjust not only the code but the comments as well :-)